tests/configuration.test.js

```diff
@@ -0,0 +1,253 @@
+'use strict';
+
+const tap = require('tap');
+const fastify = require('fastify');
+const { default: Dats } = require('@immobiliarelabs/dats');
+const sinon = require('sinon');
+const plugin = require('../');
+const { STATSD_METHODS } = require('../lib/util');
+
+async function setup(opts) {
+    const app = fastify();
+    app.register(plugin, opts);
+    app.get(
```

Comment:
Yesterday in one of my tests I've refactored all the route using `fastify.route(opts)`, if we use that approach we can externalize the `setup` function in an utily who takes an array of route configuration objects, this way we would have only one `setup` function

Reply:
it seems a good approach!